test/e2e/framework/pod/wait.go

```diff
@@ -312,6 +313,15 @@ func WaitForMatchPodsCondition(c clientset.Interface, opts metav1.ListOptions, d
 	return fmt.Errorf("gave up waiting for matching pods to be '%s' after %v", desc, timeout)
 }
 
+// GetPodDescription returns a string representing a pod in a consistent human readable format,
+// with pod UID as part of the string.
+// This should match the format defined in kubernetes/pkg/kubelet/util/format.
+func GetPodDescription(podName, podNamespace string, podUID types.UID) string {
```

Comment:
Why we need to add this function?
This addition doesn't remove import of k/k code from this package.

Reply:
see https://github.com/kubernetes/kubernetes/pull/89504#discussion_r398715368

Reply:
@tanjunchen Since b748c702d995dc4c26dfc1a12fb4beb5d8de91ae the direct import(format) has been removed already by @SataQiu . So we don't need to change here today.

Reply:
OK , I prefer to this change, revert this . Thanks

Reply:
/cc @SataQiu 
/cc @oomichi 
updated . could you approve this?Thanks